src/test/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAOSpec.scala

```diff
@@ -240,8 +241,6 @@ class PostgresDirectoryDAOSpec extends FreeSpec with Matchers with BeforeAndAfte
 
         dao.loadUser(user.id).unsafeRunSync() shouldEqual Option(user)
       }
-
-      "not delete a user's pet SA when the user is deleted" is pending
```

Comment:
we realized that this test is unnecessary because a user cannot be deleted if it still has any pets since it would throw a foreign key exception